tools/jsii-compliance/README.md

```diff
@@ -0,0 +1,65 @@
+# jsii compliance
+
+This directory contains scripts and resources to create and validate the jsii compliance suite.
+
+### Compliance Suite
+
+The compliance suite is defined as a collection of abstract test cases, that each language binding must implement individualy.
+Language specific exclusions may be added to each test separately, or to the entire suite.
+
+> See [suite.ts](./suite.ts).
+
+Note that if you add a compliance test to a specific language binding only, i.e without adding it to the suite definition, the build will fail. For example:
+
+```console
+Test 'REMOVEME' from golang report does not exist in the compliance suite. Please add it to the suite definition.
+```
+
+### Compliance Report
+
+Each language binding is responsible for creating a language specific report during the `test` phase of our build.
+
+The report takes the following form:
+
+```json
+{
+  "<test-case-name>": {
+    "status": "success | failure"
+  }
+}
+```
+
+The `<test-case-name>` must match the test case name in the suite definition. For example, the following test case definition:
+
+```json
+{
+  "name": "maps",
+  "description": "",
+  "exclusions": {}
```

Comment:
Exclusions? Shouldn't that be something reported by the language specific suite as "N/A" or is it something else?